app/src/main/java/de/westnordost/streetcomplete/quests/power_attachment/AddPowerAttachmentForm.kt

```diff
@@ -0,0 +1,13 @@
+package de.westnordost.streetcomplete.quests.power_attachment
+
+import de.westnordost.streetcomplete.quests.AImageListQuestForm
+
+class AddPowerAttachmentForm : AImageListQuestForm<PowerAttachment, PowerAttachment>() {
+
+    override val items = PowerAttachment.values().map { it.asItem() }
```

Comment:
```suggestion
    override val items = PowerAttachment.entries.map { it.asItem() }
```

this is recommended in place of values() since Kotlin 1.9. (Don't know why)